scalameta/parsers/shared/src/main/scala/scala/meta/internal/parsers/ScalametaParser.scala

```diff
@@ -196,6 +198,10 @@ class ScalametaParser(input: Input)(implicit dialect: Dialect) { parser =>
   @inline private def tryAheadNot[T: ClassTag]: Boolean =
     nextIf(!peekToken.is[T])
 
+  private def unreachable(debugges: Map[String, Any]): Nothing = UnreachableError.raise(debugges)
```

Comment:
Another set of macros that were useful, but were possible to replace

Reply:
debugges? typo?

Reply:
Fixed